OpenRA.Mods.Common/Widgets/Logic/ServerCreationLogic.cs

```diff
@@ -147,7 +148,7 @@ void CreateAndJoin()
 				return;
 			}
 
-			ConnectionLogic.Connect(IPAddress.Loopback.ToString(), Game.Settings.Server.ListenPort, password, onCreate, onExit);
+			ConnectionLogic.Connect(new ConnectionAddress(IPAddress.Loopback.ToString(), Game.Settings.Server.ListenPort), password, onCreate, onExit);
```

Comment:
This will obviously fail if the IPv4 loopback address isn't reachable. Using the hostname `localhost` should work for all cases.